examples/jenkins/master-slave/README.md

```diff
@@ -65,7 +65,7 @@ Before you begin, ensure you have created the [default imagestreams](https://doc
    ready to be used.
 
 8. Now click on *Add to project* again and select the `jenkins-master` template.
-   Here you can pick your Jenkins administrator password, alternative Docker
+   Here you can pick your authentication plugin and if needed Jenkins administrator password, alternative Docker
```

Comment:
there are significant differences ... the master has a build config for example

i'll remove the pick your password ... picking the auth plugin should also be removed, since it is hard coded per earlier discussion on use of env var.